code/Rmd/CountMatrixmiRNA.Rmd

```diff
@@ -0,0 +1,127 @@
+---
+title: "miRNA descriptive analysis"
+author: "Lorena Pantano"
+date: "02/09/2015"
+output: html_document
+---
+
+```{r libs, warning=FALSE, message=FALSE}
+library(tidyr)
+library(dplyr)
+library(knitr)
+library(synapseClient)
+library(DESeq2)
+library(vcd)
+# library(CHBUtils)
+library(corrplot)
+
+synapseLogin()
+
+options(xtable.type="html")
+
+knitr::opts_chunk$set(
+  echo=TRUE,
+  warning=FALSE,
+  message=FALSE,
+  error = FALSE,
+  tidy = FALSE)
+```
+
+# Load Data
+
+## Get metadata and count matrix 
+
+```{r get-id}
+Query <- "select id,UID,name,fileType,dataType from file where name=='Summarized miRNA Counts' AND  projectId=='syn1773109'"
+res <- synQuery(Query)
+mat_file = synGet(res$file.id)
+
+Query <- "select id,UID,name,fileType,dataType from file where name=='miRNA metadata' AND  projectId=='syn1773109'"
+res <- synQuery(Query)
+meta_file = synGet(res$file.id)
+
+```
+
+
+## Clean and sync tables
+
+```{r getcountmat}
+meta = read.table(meta_file@filePath,header=1,row.names=1,sep="\t")
+row.names(meta) = gsub("-",".",row.names(meta))
+mat = read.table(mat_file@filePath,header=T,row.names=1,sep="\t")
+
+meta_clean = meta[ row.names(meta) %in% colnames(mat), ]
+mat = mat[,as.character(row.names(meta_clean))]
+mat[is.na(mat)] = 0
+```
+
+# Results
+
+## Correlation of metadata variables
+
+```{r cor}
+get_cor = function(v){
```

Comment:
really useful; we should make a place for libs/convenience functions, like what your `CHBUtils` is doing (or, you can add there, annotate it)